### ISSUE TITLE
[NPC][Skinning][Loot] Anub´ar Underlord/Slayer's give no Nerubian Chitin

### ISSUE THREAD

**Issue by balkai** [dalaranwow/dalaran-wow#3984]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Mobs do not drop Nerubian Chitin when skinned.
**Expected behaviour**: Tell us what should happen instead
Nerubian Chitin should drop when skinned.
**Steps to reproduce the problem**:

1.  Kill mob
2.  Loot mob
3.  Skin, no chitin

**Include proofs for this behaviour**
Wowhead loot tables say droprate should be 20% via skinning, I've killed well over 50 and no chitin.
You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
Anub´Ar Underlord ID: 26605
Anub´Ar Slayer ID: 26606
You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
DB/Loot: Add Chitrin lott (1-5) for 20 percent to creatures: Nerub\'ar Web Lord,Nerub\'ar Warrior,Nerub\'ar Tunneler,Anub\'ar Ambusher,Anub\'ar Dreadweaver,Anub\'ar Underlord,Anub\'ar Slayer

* closes https://github.com/dalaranwow/dalaran-wow/issues/3984
* commit: f793c2c90a4e63959d459f9242333d66779adb11

The issue was fixed by a developer and will now be tested on the Public Test Realm (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Only once the issue was marked with "Fixed - On Live Server", you can find the fix on you server.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.
